### PR TITLE
239 update cardcreator

### PR DESCRIPTION
**Pull request by lxgr-linux**
dont't merge

### CODE REVIEW

File: src/views/CardCreatorPage.vue [hunk 1]
```diff
@@ -847,6 +829,9 @@ export default {
     }
   },
   methods: {
+    isEmpty (a) {
```
Comment (lxgr-linux) [inline src/views/CardCreatorPage.vue:832]: I dont think this is necessary

Comment (patrickwieth) [reply src/views/CardCreatorPage.vue:832]: It is used in L554 
https://github.com/DecentralCardGame/frontend/pull/265/files#diff-048c994414ac950e6ee9aa6f7f6a1278da659f026c01c550027aaf0c71be70beR554

Before it was changed to !abilities which did not work properly

**Comment by patrickwieth** [comment]
PR can be merged if review succesful

File: src/views/CardCreatorPage.vue (hunk 1 above)
Comment (lxgr-linux) [reply src/views/CardCreatorPage.vue:832]: I mean the wrapper might not be necessary

**Comment by lxgr-linux** [comment]
image
image

Can u check this out?

**Comment by patrickwieth** [comment]
website and blockchain have the same cardobject?

**Comment by patrickwieth** [comment]
error does not come from cardobject mismatch

variables in mill are not properly set

**Comment by patrickwieth** [comment]
The Bug comes from removing the emit on the DropDown component that allowed @change="showAbilityModal(...)"

**Comment by patrickwieth** [comment]
Merge this or god kills a kitten!

**Pull request merged**